app/core/Engine/Engine.ts

```diff
@@ -934,7 +934,9 @@ export class Engine {
 
     const requireAllowlist = process.env.METAMASK_BUILD_TYPE === 'main';
     const disableSnapInstallation = process.env.METAMASK_BUILD_TYPE === 'main';
```

Comment:
```suggestion
    const disableSnapInstallation = process.env.METAMASK_BUILD_TYPE !== 'flask';
```
I would assume we don't want external Snaps be installable in beta?